defs/container.bzl

```diff
@@ -0,0 +1,53 @@
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_image",
+    "container_push",
+)
+load(
+    "@io_bazel_rules_docker//container:bundle.bzl",
+    "container_bundle",
+)
+
+# image macro creates basic image and push rules for a main
+def image(binary, visibility = ["//visibility:public"]):
+    if len(binary) == 0:
+        fail("binary is a required argument")
+    if binary[0] != ":":
+        fail("binary must be a package local label")
+    name = binary[1:]
+    image = name + "-image-internal"
+    container_image(
+        name = image,
+        entrypoint = ["/" + name],
```

Comment:
I changed this from "cmd" to "entrypoint" which is potentially a better default since the base image is basically scratch, but it will probably break the gcp-controller-manager deployment once we upgrade it.

Reply:
Can we change this in a separate PR?

Reply:
Done.